src/res/globalStyle.css

```diff
@@ -149,7 +149,6 @@ select {
 
 .u-form .u-input,
 .u-form textarea {
-    border: 1px solid;
```

Comment:
Removing this border makes it very unclear where input boxes are now
![image](https://user-images.githubusercontent.com/725880/56585314-7eb8f200-65d5-11e9-91a0-4ef0c52c3bae.png)